src/examples/rest/src/backend/schema/auth/password/entity.ts

```diff
@@ -5,6 +5,7 @@ import {
 } from '@exogee/graphweaver-auth';
 import { Credential as OrmCredential } from '../../../entities';
 
+// I don't think this Credential type is correct
```

Comment:
This is grabbing the [Web API Credential](https://developer.mozilla.org/en-US/docs/Web/API/Credential), not sure how to type this.